config/tectonic-node-controller/config.go

```diff
@@ -19,7 +19,7 @@ const (
 // ControllerConfig is the config structure TNC to generate NodeConfigs
 type ControllerConfig struct {
 	metav1.TypeMeta     `json:",inline"`
```

Comment:
Hey @abhinavdahiya just to clarify, fetch command is not currently being used. kubelet-Kubeconfig it's embed into the ignition user-data config https://github.com/coreos/tectonic-installer/blob/master/installer/pkg/config-generator/ignition.go#L59
The kubeconfig-kubelet needs some "dynamic" parameters, e.g:
`${root_ca_cert}`
`${kubelet_bootstrap_token_id}.${kubelet_bootstrap_token_secret}`
How are planning to populate the kubeconfig generated by the tnc with those?

Reply:
@enxebre 
Well the plan is
** tnc bootstrap will serve bootstrap kubeconfig from on-disk kubeconfig
** tnc Daemonset will serve bootstrap kubeconfig based on serviceaccount in the cluster.